engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/layout/emitter/IPage.java

```diff
@@ -50,11 +58,36 @@ public interface IPage {
 	 */
 	void drawText(String text, int textX, int textY, int width, int height, TextStyle textStyle);
 
+	/**
+	 * Draw the image
+	 *
+	 * @param imageId   image id
+	 * @param imageData image data
+	 * @param extension image type
+	 * @param imageX    image x position
+	 * @param imageY    image y position
+	 * @param height    image height
+	 * @param width     image width
+	 * @param helpText  help text
+	 * @throws Exception handling exception
+	 */
 	void drawImage(String imageId, byte[] imageData, String extension, int imageX, int imageY, int height, int width,
```

Comment:
Do you think we can break this API?

Reply:
@wimjongman 
Of course a good question.
The point is that the parameter won't be used on the methods. But I have no problem if we decide to undo the change.

Your favorit would be to leave it like it is - correct?

Reply:
Yes. This is a public API. Other implementations could use the map.

Reply:
Ok, I have removed the changes of that.
You see it on the details and also on the changed files.